databases/week1/homework.sql

```diff
@@ -0,0 +1,38 @@
+-- Active: 1719914052416@@0.0.0.0@3307@sales
+
+-- 1. Find out how many tasks are in the task table
+
+SELECT COUNT (*) FROM task
+
+-- 2. Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(due_date) FROM task
+
+-- 3. Find all the tasks that are marked as done
+SELECT status.name, task.title FROM (task INNER JOIN status ON status.name='done' )
+
+-- 4. Find all the tasks that are not marked as done
+SELECT user.name , status.name from (user inner join status) WHERE status.name !='done'
+ 
+ --5. Get all the tasks, sorted with the most recently created first
+SELECT * from task ORDER BY created DESC
+
+-- 6. Get the single most recently created task
+SELECT * from task ORDER BY created DESC LIMIT 1
+-- 7. Get the title and due date of all tasks where the title or description contains database
+SELECT title,due_date from task WHERE task.title !='null' AND task.description !='null'
```

Comment:
Here you need to get all tasks whose title or description contain "database", you need to change your conditional, here you are getting all titles and descriptions that are not null.